src/com/ableton/PythonBuilder.groovy

```diff
@@ -0,0 +1,101 @@
+package com.ableton
+
+
+class PythonBuilder implements Serializable {
+  @SuppressWarnings('FieldTypeRequired')
+  def script = null
+  /**
+   * Python major version to download (required value). Example: '3.7.0'
+   */
+  String version = null
+
```

Comment:
Style nitpicking, why is there an empty here?

Reply:
I wanted to differentiate between the required and optional parameters.

Reply:
I see. Then let's leave it here.